assets/training/distillation/components/pipeline/spec.yaml

```diff
@@ -257,7 +259,7 @@ jobs:
 
   oss_distillation_generate_data:
     type: command
-    component: azureml:oss_distillation_generate_data:0.0.5
+    component: azureml:oss_distillation_generate_data:0.0.6
```

Comment:
we recently faced some issues with versioning , the correct steps to follow would be:
1. just before release, check the version in azureml registry, and make the increment for all components/pipelines which will be updated due to the release.
2. build all the affected components with the updated versions, release them together.

not immediately relevant to the PR, but will become important during release.

Reply:
Sounds good, will verify all versions in prod after this PR